Modules/BenchPress.Azure/Public/Confirm-StorageContainer.ps1

```diff
@@ -44,13 +44,13 @@ function Confirm-StorageContainer {
     [string]$ResourceGroupName
   )
   Begin {
-    $ConnectResults = Connect-Account
+    $connectResults = Connect-Account
   }
   Process {
-      $Resource = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName -AccountName $AccountName `
-       | Get-AzStorageContainer -Name $Name
+      $resource = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName -AccountName $AccountName
+        | Get-AzStorageContainer -Name $Name
```

Comment:
```suggestion
       | Get-AzStorageContainer -Name $Name
```

For multi-line pipes, I've been removing the first space on the new line

Reply:
I'm down for consistency; I can't speak to other files that are in the system, but all the example test files use the no-indent currently so I'm okay keeping as is. We'd have to change all of those

Reply:
You beat me to it. I realized the ApiManagement files have indenting and started searching for any best practice recommendations. In order to align with the `examples` directory, we can go no-indent? If there are no objections, I can go through the cmdlets and remove indentation in this PR as well.

Reply:
@ozboms when you say "keeping as is", do you mean keeping the example test files as is? So going no indent?

Reply:
Sounds like no-indent is the path ahead. Yes please @msha1026 , do go through and update them in the examples and in the `.ps1` files.

Reply:
Yep!

Reply:
Alright I have applied the no-indent route to all files.